lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -681,6 +683,26 @@ func (c *ContainerProxy) StreamContainerStats(ctx context.Context, config *conve
 	return nil
 }
 
+func (c *ContainerProxy) GetContainerChanges(vc *viccontainer.VicContainer) (io.ReadCloser, error) {
```

Comment:
this is the interface of ContainerProxy, not Docker API, that's why I want to have one more parameter here, so that I can get data back. I did that change in my PR #5719